instrumentation/src/javatests/org/oppia/android/instrumentation/application/EndToEndTestGcsResourceModuleTest.kt

```diff
@@ -0,0 +1,93 @@
+package org.oppia.android.instrumentation.application
+
+import android.app.Application
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.android.util.gcsresource.DefaultResourceBucketName
+import org.oppia.android.util.gcsresource.QuestionResourceBucketName
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(application = EndToEndTestGcsResourceModuleTest.TestApplication::class)
+class EndToEndTestGcsResourceModuleTest {
+
+  @Inject
+  @DefaultResourceBucketName
+  lateinit var defaultResourceBucketName: String
+
+  @Inject
+  @QuestionResourceBucketName
+  lateinit var questionResourceBucketName: String
+
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+  }
+
+  @Test
+  fun testModule_getDefaultGcsResource_returnsAssetsDevHandler() {
```

Comment:
```suggestion
  fun testModule_defaultGcsResource_isAssetsDevHandler() {
```

Ditto for the other module tests here & in the other suite.

Reply:
Done